cookiecutter.json

```
@@ -40,7 +42,8 @@
         "license": "Select a license",
         "backend": {
             "__prompt__": "Choose a build backend",
-            "hatch": "Hatchling (recommended for pure Python projects)"
+            "hatch": "Hatchling (recommended for pure Python projects)",
+              "vcs": "Enable hatch version control (vcs)?"
```

I'm not sure if non-experienced packagers will understand what "Enable hatch version control (vcs)?" means. I propose changing it to this:


```suggestion
            "hatch": "Hatchling (recommended for pure Python projects)",
              "vcs": "Let hatch get the version from your version control system (Git/Mercurial/SVN)?"
```